pkg/api/live/v1/interfaces/types.go

```diff
@@ -60,15 +60,16 @@ type MessageResponse struct {
 
 // MetadataResponse is the response from a live transcription
 type MetadataResponse struct {
-	Channels       int                  `json:"channels,omitempty"`
-	Created        string               `json:"created,omitempty"`
-	Duration       float64              `json:"duration,omitempty"`
-	ModelInfo      map[string]ModelInfo `json:"model_info,omitempty"`
-	Models         []string             `json:"models,omitempty"`
-	RequestID      string               `json:"request_id,omitempty"`
-	Sha256         string               `json:"sha256,omitempty"`
-	TransactionKey string               `json:"transaction_key,omitempty"`
-	Type           string               `json:"type,omitempty"`
+	Channels       int                    `json:"channels,omitempty"`
+	Created        string                 `json:"created,omitempty"`
+	Duration       float64                `json:"duration,omitempty"`
+	ModelInfo      map[string]ModelInfo   `json:"model_info,omitempty"`
+	Models         []string               `json:"models,omitempty"`
+	RequestID      string                 `json:"request_id,omitempty"`
+	Sha256         string                 `json:"sha256,omitempty"`
+	TransactionKey string                 `json:"transaction_key,omitempty"`
+	Type           string                 `json:"type,omitempty"`
+	Extra          map[string]interface{} `json:"extra,omitempty"`
```

Comment:
I think this should just be `string` like python?
https://github.com/deepgram/deepgram-python-sdk/pull/274/files#diff-401c5426b08bd5ff5ab0b206b8f4cf3a55cc10b70c1b5cdd84dc471bdcd054f5R150

If not, `interface{}` should be avoid here.

Reply:
When I tried it with `string`, it did not work when I ran the example. But when I tried it with `map[string]interface{}`, it did work. Originally, I thought there was something wrong because I would test the examples and extra never showed up. So then I looked at the Node SDK and I saw that in the Node SDK, the type is this

```
extra: {
    [key: string]: unknown;
  };
```
And when you run it in the Node SDK, the result is this: `extra: { test: 'one' },`

So that made me think that the result shouldn't be a string, it should be an object. Is there a better way to type the result as an object in Go?

Reply:
It should be:
```
type Metadata struct {
        ...
	Extra          map[string]string    `json:"extra,omitempty"`
        ...
}
```

and it looks like what is there for the input/options is wrong and should be:
```
type PreRecordedTranscriptionOptions struct {
        ...
        Extra            []string `json:"extra,omitempty" url:"extra,omitempty"`
        ...
}
```

Reply:
So I changed the type to `map[string]string` like you suggested and now it works. I don't think we need to change the input/options to `[]string`, we just leave that as `string`. This allows for the user to add the options as:

```
options := interfaces.PreRecordedTranscriptionOptions{
		Extra:      "one:two",
	}
```

And the result they get is

```
"extra": {
      "one": "two"
    },
```